tutorials/Threshold_Weighted_Scores.ipynb

```diff
@@ -8,7 +8,7 @@
     "## Introduction\n",
     "For a long time, people have wanted to be able to evaluate the performance of forecasts of extreme events using verification methods that can't be hedged. The threshold weighted scores in `scores` allow us to do this using consistent scoring functions.\n",
     "\n",
-    "Threshold weighted scores allow us to measure the performance of forecasts with a focus on a range of decision thresholds (e.g., performance of temperature forecasts above 40°C) using [consistent scoring functions](./Consistent_Scores.ipynb) and they do not allow hedging. They allow us to measure the performance of predicting extremes or some other range of decision thresholds of interest. This is particularly important when common mean scoring functions such as the [mean squared error (MSE)](./Mean_Squared_Error.ipynb) evaluate performance across all decision thresholds may hide the performance of predicting extremes.\n",
+    "Threshold weighted scores allow us to measure the performance of forecasts with a focus on a range of decision thresholds (e.g., users who key decision thresholds are above 40°C) using [consistent scoring functions](./Consistent_Scores.ipynb) and they do not allow hedging. They allow us to measure the performance of predicting extremes or some other range of decision thresholds of interest. This contrasts with metrics like mean squared error (MSE) and mean absolute error (MAE), which can be interpreted as evaluating performance when equal weight is placed on all decision thresholds ([Ehm 2016](https://doi.org/10.1111/rssb.12154)).\n",
```

Comment:
I am a bit confused by the phrase "(e.g., users who key decision thresholds are above 40°C)". Do you mean something along the line of: "users whose key decision thresholds are temperatures above 40°C" or "users with a key decision threshold of temperatures above 40°C"?

Reply:
Thanks, I've added the word "temperature" for clarity

Reply:
This may be me misunderstanding, but is the "who" meant to be a "whose"? 

E.g. "users whose key decision thresholds are temperatures above 40°C"

Reply:
Yes, that's exactly what I meant to write. I'll update it now.